oas_worker/evaluate_devset.py

```diff
@@ -0,0 +1,379 @@
+#!/usr/bin/env python
+""" evaluate_devset
+
+Evaluate OAS Devset:
+  * Keywords
+  * Transcripts (TBD)
+
+Author(s): flipsimon, datadonk23
+Date: 16.12.21
+"""
+
+import httpx
+import json
+import argparse
+import pandas as pd
+import statistics
+from nltk.stem.cistem import Cistem
```

Comment:
why you use the nltk cistem stemmer instead of the spacy stemmer? So far we have used Spacy, if there is no reason to use NLTK, I would like to avoid additional dependencies.

Reply:
Because the NLTK stemmer is just simple (but powerful enough for the moment) to use. For spacy, you would need to load the model again (for just simple stemming), which slows things down as you can see when spacy_pipe is used. And there is no stemmer in spacy, at least afaik, only a lemmatizer.
But, can try a version using spacy lemmatizer when writing the worker jobs for the full evaluation next. There we can see, how things are getting slowed downed actually and what's the impact on evaluation results eventually. Then we can decide, if adding the dependency is worth or not.

Reply:
FYI: Switched to spacy lemmatization. It actually slows things down a bit (2,5s for our 20 samples, compared to using Cistem by nltk), but that's not a big deal compared to overall runtime. So from my point of view, it's not worth bringing in a new dependency (at least for the moment).